{"name": "# C5176 Add a new device", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <span _ngcontent-sfu-c8=\"\">...</span> is not clickable at point (1188, 77). Other element would receive the click: <div _ngcontent-sfu-c8=\"\" class=\"spinner-outer ng-star-inserted\">...</div>\n  (Session info: chrome=86.0.4240.111)\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5176_Add_a_new_device.py\", line 16, in clck_add_nw_dvc\n    context.app.main_page.clck_add_nw_dvc()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 1041, in clck_add_nw_dvc\n    wait.until(EC.element_to_be_clickable(ADD_NW_DVC_RGHT_CRNR)).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1604156399676, "stop": 1604156402687}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1604156402687, "stop": 1604156405922}, {"name": "And Go to the Devices page https://devcloud.connectedio.com/devices", "status": "passed", "start": 1604156405923, "stop": 1604156406161}, {"name": "Then Click Add new device button on the right up corner", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <span _ngcontent-sfu-c8=\"\">...</span> is not clickable at point (1188, 77). Other element would receive the click: <div _ngcontent-sfu-c8=\"\" class=\"spinner-outer ng-star-inserted\">...</div>\n  (Session info: chrome=86.0.4240.111)\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5176_Add_a_new_device.py\", line 16, in clck_add_nw_dvc\n    context.app.main_page.clck_add_nw_dvc()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 1041, in clck_add_nw_dvc\n    wait.until(EC.element_to_be_clickable(ADD_NW_DVC_RGHT_CRNR)).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1604156406161, "stop": 1604156407302}, {"name": "Then Fill required Device IMEI field 356961071557824", "status": "skipped", "start": 1604156409427, "stop": 1604156409427}, {"name": "And Fill Device Name field if any TVM Test123 jjnkjn", "status": "skipped", "start": 1604156409427, "stop": 1604156409427}, {"name": "Then Fill required Serial Number field 1806208A0279", "status": "skipped", "start": 1604156409427, "stop": 1604156409427}, {"name": "And Click Save and refresh the page", "status": "skipped", "start": 1604156409427, "stop": 1604156409427}, {"name": "Then Success is here - device is on the page", "status": "skipped", "start": 1604156409427, "stop": 1604156409427}, {"name": "Then Delete device", "status": "skipped", "start": 1604156409427, "stop": 1604156409427}], "start": 1604156394357, "stop": 1604156409427, "uuid": "0ef1e247-92a0-46b7-8e62-ad6d341ac218", "historyId": "f66297f0cc4ca767f9caf1684373e3ca", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5176 Add a new device"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}